com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/ShaderLibrary/Input.hlsl

```diff
@@ -90,7 +90,8 @@ CBUFFER_END
 #define UNITY_MATRIX_I_M   unity_WorldToObject
 #define UNITY_MATRIX_I_P   (float4x4)0
 #define UNITY_MATRIX_I_VP  (float4x4)0
-#define UNITY_PREV_MATRIX_I_M unity_MatrixPreviousMI
```

Comment:
Should these PREV matrices just be defined to the current ones instead? This way things should kind of work (I think) instead of not at all?